### PR TITLE
Total added to home page

### PR DESCRIPTION
**Pull request by donchuru**
The total estimate feature is not entirely ready but i'm merging so that everyone has the same-looking home page.

### CODE REVIEW

**Comment by donchuru** [comment]
The branch will stay open until the feature fully works

**Review by ijoffe — CHANGES_REQUESTED** [review]
Please delete

EDIT: Added this by mistake

File: InventoryManager/app/src/main/java/com/example/inventorymanager/ui/home/HomeFragment.java
Comment (ijoffe) [inline InventoryManager/app/src/main/java/com/example/inventorymanager/ui/home/HomeFragment.java:1]: Please remove lines 48-50

Comment (ijoffe) [reply InventoryManager/app/src/main/java/com/example/inventorymanager/ui/home/HomeFragment.java:1]: lgtm now

**Pull request merged**